opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -244,20 +244,59 @@ def force_flush(self, timeout_millis: int = 30000) -> bool:
         return True
 
 
+# skip natural LogRecord attributes
+# http://docs.python.org/library/logging.html#logrecord-attributes
+_RESERVED_ATTRS = frozenset(
+    (
+        "asctime",
+        "args",
+        "created",
+        "exc_info",
+        "exc_text",
+        "filename",
+        "funcName",
+        "getMessage",
+        "levelname",
+        "levelno",
+        "lineno",
+        "module",
+        "msecs",
+        "msg",
+        "name",
+        "pathname",
+        "process",
+        "processName",
+        "relativeCreated",
+        "stack_info",
+        "thread",
+        "threadName",
+    )
+)
+
+
 class OTLPHandler(logging.Handler):
     """A handler class which writes logging records, in OTLP format, to
     a network destination or file.
     """
 
-    def __init__(self, level=logging.NOTSET, log_emitter=None) -> None:
+    def __init__(
+        self,
+        level=logging.NOTSET,
+        log_emitter=None,
+    ) -> None:
         super().__init__(level=level)
         self._log_emitter = log_emitter or get_log_emitter(__name__)
 
+    @staticmethod
+    def _get_attributes(record: logging.LogRecord) -> Attributes:
+        return {
+            k: v for k, v in vars(record).items() if k not in _RESERVED_ATTRS
```

Comment:
Is `extra` the only keyword that will be filtered in?

Reply:
Yes, that's how the stdlib logging library works.

Documentation: https://docs.python.org/3/library/logging.html#logging.Logger.debug (paragraph 2)

Source: https://github.com/python/cpython/blob/80f33f266b4ad5925a3e58ea3a54ae139a6b6f0e/Lib/logging/__init__.py#L1597-L1601